en/docs/ipaas/micro-integrator/develop-integrations-with-integration-studio.md

```diff
@@ -59,7 +59,11 @@ During deployment, Choreo performs a Trivy scan to detect critical vulnerabiliti
 
 You can use environment variables in WSO2 Micro Integrator to improve the configuration management, security, portability, and manageability of the Micro Integrator installation. By using environment variables, organizations can simplify the management and maintenance of their micro installations and ensure that they can be quickly and easily updated when required.
 
-The Choreo DevOps Portal allows you to define configurations and secrets to a deployed integration (as well as other component types). For more information, see [Configurations and secrets](../../devops/devops-portal.md#configurations-and-secrets).
+The Choreo Console allows you to define enviornment variables for your integration project. You can define the environment variables before you deploy the component.
```

Comment:
```suggestion
The Choreo Console allows you to define environment variables for your integration project. You can define the environment variables before you deploy the component.
```

Reply:
Fixed in fd88d99a57c17c5241a87960c3e13d754445543a